Dockerfile

```diff
@@ -40,10 +40,8 @@ RUN apk add --no-cache libgmpxx
 
 COPY --from=cairo-lang /wheels /wheels
 
-COPY --from=compiler-builder /cairo/corelib /usr/local/bin/corelib
-COPY --from=compiler-builder /cairo/target/release/starknet-sierra-compile /usr/local/bin/target/release/starknet-sierra-compile
-COPY --from=compiler-builder /cairo/target/release/starknet-compile /usr/local/bin/target/release/starknet-compile
-COPY --from=compiler-builder /root/.local/share/scarb-install/${SCARB_VERSION}/bin/scarb /usr/local/bin/scarb
+COPY --from=compiler /cairo /usr/local/bin/cairo
```

Comment:
Won't this break compatibility with starknet-hardhat-plugin? It's expecting the binaries to be in /usr/local/bin/target/release (defined in constants.ts). Let's keep it backwards compatible, but leave a comment in this part of the file.

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/19/commits/495d1553932745eef246d6013af98c9ca55441af

Reply:
I meant a comment which explains why we are copying it to that exact directory (to keep it compatible with what the plugin is expecting).

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/19/commits/495d1553932745eef246d6013af98c9ca55441af